deploy/examples/csi/rbd/storageclass-ec.yaml

```diff
@@ -59,6 +59,16 @@ parameters:
   # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
   # unmapOptions: force
 
+   # (optional) Instruct the plugin it has to encrypt the volume
```

Comment:
IMO, this reads a little bit funny. It could also be a little more clear for users. I would suggest something like, "encrypt each volume with encryption keys from a key management system (KMS)."

Or am I misunderstanding something and encryption can be enabled without a KMS connection?